### ISSUE TITLE
The focus is not on the modal when it is opened by keyboard navigation

### ISSUE THREAD

**Issue by hannyle** [CSCfi/swift-browser-ui#914]
#### Describe the bug

When using keyboard to open a modal, the modal is open but the focus is on another element, not on the modal

#### To Reproduce
Steps to reproduce the behaviour:
1. Go to SD Connect's browser page
2. Press button `Create Folder` to open modal
3. The focus is not on Create Folder modal, but on the `Upload` button instead

#### Expected behavior

The focus should be right on modal whenever it's opened
